.github/workflows/tests.yml

```diff
@@ -83,3 +83,7 @@ jobs:
       - name: Hello World tests
         run: |
           cd hello-world && yarn test
+
+      - name: Gated NFT tests
+        run: |
+          cd gated-nft/zksync && yarn install --frozen-lockfile && yarn test
```

Comment:
you don't need `yarn install --frozen-lockfile` as [deps are installed from the root directory for all tutorials using yarn workspaces](https://github.com/matter-labs/tutorials/blob/ed847d9a34677af74f519a548a4b63017f07e18d/package.json#L4-L16)